ONSdigital/eq-survey-runner
# implemented census header and footer

### What is the context of this PR?

[Trello](https://trello.com/c/K7O3MTie/682-census-header-footer-design-8)

Implements header and footer design for the census theme, including hidden "help and support" UI across mobile, tablet and desktop sizes.

- Clicking the "help and support" button should reveal the help panel, clicking it again to close.
- Tabbing through the page should reveal the help panel when focus lands on a link inside it.

### How to review 

- Start `census_theme` schema
- Confirm design is correct as per attachments on Trello card across all screen sizes
- Confirm "Help and support" button works
- Confirm using tab to navigate through page reveals help and support panel
